tensorboard/plugins/image/summary_v2.py

```diff
@@ -27,6 +27,43 @@
 def image(name, data, step=None, max_outputs=3, description=None):
     """Write an image summary.
 
+    See also `tf.summary.scalar`, `tf.summary.SummaryWriter`.
+
+    Writes a collection of images to the current default summary writer. Data
+    appears in TensorBoard's 'Images' dashboard. Like `tf.summary.scalar` points,
+    each collection of images is associated with a `step` and a `name`.  All the
+    image collections with the same `name` constitute a time series of image
+    collections.
+
+    This example writes 2 random grayscale images:
+
+    ```python
+    w = tf.summary.create_file_writer('test/logs')
```

Comment:
Hey, for these examples, can you consider making these testable?  See: https://www.tensorflow.org/community/contribute/docs_ref for details.

Reply:
I updated it to use "doctest style" `>>>`, `...`s, but I can't seem to get the test to run at all.

Digging into `bazel run //tensorflow/tools/docs:tf_doctest`, it seems that passing `--file` only works when the file is in the tensorflow repo, which TensorBoard's V2 summaries are not.  Passing `--module` and building from source didn't work for me, as I got some unrelated mlir error during the bazel build.

I did do a test sync cl/364014007 and saw that TAP presubmit passed.  Is this sufficient?

Reply:
I had a quick look at this and confirmed @psybuzz's findings that there's no way to run it using the current `--file` or `--module` options. 

It looks like those two interactive options could be relatively simple to fix.

But, AFAIKT, it still wouldn't run as an actual test. The main  reason is that it's setup to find modules within `tensorflow.python`, so I don;t think tf_doctest would find this.

I think the least resistance way to get this working would be to copy tf_doctest into your package, like they did with [tf_estimator_doctest.py](https://github.com/tensorflow/estimator/blob/3fb3b55b05b3eb75f3ad284b010f964796ef7cae/tensorflow_estimator/python/estimator/tf_estimator_doctest.py)

>  cl/364014007  and saw that TAP presubmit passed. Is this sufficient?

I don't see the doctest in the list of tests that ran. Since TensorFlow's a separate project, this probably has something to do with the blueprint setup. If it had run, don't think it would test this, so that's fine.

Reply:
Thanks for confirming!  For now, I would like to keep the scope of these PRs on the content and structure of the docs, and make the task of improving our infra by introducing TensorBoard doctests a separate PR.  These doctests, if run, would actually change the filesystem which is not ideal, so we might want to consider a doctest setup that mocks out filesystem effects.

I wasn't aware of tf_estimator_doctest.py, it's nice to know another team has done this pattern.